scripts/start-baker.sh

```diff
@@ -13,19 +13,7 @@ gen_node_identity() {
 }
 
 start_node() {
-    case "$base_chain" in
-        babylonnet )
-            network="babylonnet"
-            ;;
-        carthagenet )
-            network="carthagenet"
-            ;;
-        *)
-            echo "$base_chain not supported. Only 'babylonnet' and 'carthagenet' are supported."
-            exit 1
-            ;;
-    esac
-    node_args=("--data-dir" "$node_dir" "--rpc-addr" "localhost:8732" "--net-addr" "$net_addr" "--no-bootstrap-peers" "--bootstrap-threshold" "1" "--network" "$network")
+    node_args=("--data-dir" "$node_dir" "--rpc-addr" "localhost:8732" "--net-addr" "$net_addr" "--no-bootstrap-peers" "--bootstrap-threshold" "1")
```

Comment:
Do we not need $network now?

Reply:
no, we provide network information in config file now